src/OpenTelemetry.Exporter.Prometheus/PrometheusExporterExtensions.cs

```diff
@@ -93,6 +106,21 @@ private static void WriteSum(StreamWriter writer, PrometheusMetricBuilder builde
             builder.Write(writer);
         }
 
+        private static void WriteGauge(StreamWriter writer, PrometheusMetricBuilder builder, IEnumerable<KeyValuePair<string, object>> labels, double doubleValue)
```

Comment:
Does this method only take a `double` value? Do we need a method that accepts `long` as well?

Reply:
Prometheus exposition format seem to be only accepting float/double, so casting long to double for now.